### PR TITLE
#20 Add class description to display of DataFrame

### LINKED ISSUE #20: Request: enhance display of DataFrame
It would be useful if some more meta data is displayed by DataFrame disp()
- Class type (eg DataFrame or TimeFrame)
- If it is a series (ColSeries or RowSeries)

### PR DESCRIPTION
**Pull request by mreijnde**
Minor change to add class to display of DataFrame and show if it is a series or not.

closes #20

### CODE REVIEW

File: +frames/DataFrame.m [hunk 1]
```diff
@@ -1380,7 +1380,20 @@ function disp(obj)
             maxCols = 50;  % Matlab struggles to show many columns
             if all(size(obj) < [maxRows,maxCols])
                 try
-                    disp(obj.t);
+                    % show content
+                    disptable = obj.t;
+                    if obj.colseries
+                       disptable.Properties.VariableNames{1} = 'NaN';
```
Comment (benjamingaudin) [inline +frames/DataFrame.m:1386]: 'missing' instead of 'NaN' as the value could be a NaT, a string(missing) as well

**Comment by benjamingaudin** [comment]
What I mean is: showing ‘NaN’ will make the user believe that the type of the missing value is a double (or a duration); while the type of the missing value could be a datetime (NaT) or a string (<missing>). So the generic name would be to show ‘missing’ so that no assumption is made for the value type. 

Note that the index (RowProperties/RowTime) of a time/table is able to show NaN or NaT, so it could be nice to show missing double, duration, and datetime as such.

**Comment by mreijnde** [comment]
I guess this is what you had in mind? The display of a series now displays the singleton dimension as it would display the missing value from the given index datatype.  I am not sure if this add much information, as the datatype of the singleton index is not used anyway.

**Comment by mreijnde** [comment]
Moved the code to handle missing value as index/column to gettable code. This also fixes bug that table conversion did not work for rowSeries().

Note: the behavior of gettable() is now changed. Conversion to valid variable names for columns is not needed and removed. Numerical column names are now displayed without the "x" prefix.

**Pull request merged**